### PR TITLE
devel-guide: Changing private-related references on text

### PR DESCRIPTION
**Pull request by eduser25**
- Removed private repo references as we are no longer private
- Updated the references to properly explain some of the container registry constraints/requirements
- Created Issue to track potential improvement on script to not depend on a private container registry

### CODE REVIEW

File: docs/development_guide.md [hunk 1]
```diff
@@ -131,9 +133,10 @@ $ # Create docker secret in Kubernetes Namespace using following script:
 $ ./scripts/create-container-registry-creds.sh "$K8S_NAMESPACE"
 
 ```
+(NOTE: these requirements are true for automatic demo deployment using the available demo scripts, #1416 tracks an improvement to not strictly require this and use upstream images from official dockerhub registry if a user does not want/need changes on the code)
```
Comment (eduser25) [inline docs/development_guide.md:136]: s/this/these

Comment (eduser25) [reply docs/development_guide.md:136]:
```suggestion
(NOTE: these requirements are true for automatic demo deployment using the available demo scripts, #1416 tracks an improvement to not strictly require these and use upstream images from official dockerhub registry if a user does not want/need changes on the code)
```

Comment (eduser25) [inline docs/development_guide.md:136]:
```suggestion
(NOTE: these requirements are true for automatic demo deployment using the available demo scripts, #1416 tracks an improvement to not strictly require these and use upstream images from official dockerhub registry if a user does not want/need changes on the code)
```

Comment (eduser25) [reply docs/development_guide.md:136]: hm... guess with the new remote fork workflow we lost he ability to commit on the local pr branch. Is it normal to not be able to commit suggestions?

image

Comment (bridgetkromhout) [reply docs/development_guide.md:136]: Right, we should not using branches in the repo at this point. We should PR from our forks.

Comment (shashankram) [reply docs/development_guide.md:136]: +1. @eduser25 please create a PR from a fork.

**Comment by bridgetkromhout** [comment]
Hey folks, let's re-open this from a fork. Thanks!

**Pull request closed without merging**